android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -17,4 +17,10 @@ enum class TranslationOperation {
      * A translated page should be restored.
      */
     RESTORE,
+
+    /**
+     * Fetch the languages that the translation engine supports to populate the to and from
+     * dropdowns.
+     */
+    FETCH_TO_AND_FROM_LANGUAGES,
```

Comment:
Name is a bit strange because I suspect we might also need something like a `FETCH_SETTINGS_LANGUAGES` later.

Reply:
Maybe we could just have `FETCH_LANGUAGES` and we could expand on the docs, what do you think?

Reply:
Thanks, updated, I like that idea much better, seems much less awkward!